add Community page (#26)
anz-bank/sysl-website: docs/discussions.md

---
id: discussions
title: Discussions
sidebar_label: Discussions
---

On this page we've listed some Sysl-related communities that you can be a part of; see the other pages in this section for additional online and in-person learning materials.

## Stack Overflow
Stack Overflow is a popular forum to ask code-level questions or if you're stuck with a specific error. Read through the [existing questions](https://stackoverflow.com/questions/tagged/sysl) tagged with `sysl` or [ask your own](https://stackoverflow.com/questions/ask?tags=sysl)!

## Slack

> There is a great place for discussion about best practices and application architecture as well as the future of Sysl. If you have an answerable code-level question, Stack Overflow is usually a better fit.
> 
> * [Slack online chat](https://anzoss.slack.com/messages/sysl/)

Reviewer: ```suggestion
[The Sysl Slack channel](https://anzoss.slack.com/messages/sysl/) is a great place for discussion about best practices and the future of Sysl. If you have specific questions about code, Stack Overflow is usually a better fit.
```
